tools/ci.py

```diff
@@ -476,7 +476,11 @@ def matrix(ctx: Context, distro_slug: str):
     _matrix = []
     for transport in ("zeromq", "tcp"):
         if transport == "tcp":
-            if distro_slug not in ("centosstream-9", "ubuntu-22.04-arm64"):
+            if distro_slug not in (
```

Comment:
We need to remove all of the tools changes. They are editing the jobs for the main test suite.

Reply:
If you recall, I asked you about this, and you replied yes, ubuntu-22.04 needed to be in there too.

Hence not understanding why you want to pull this unless in its own PR ?